package.json

```diff
@@ -5,20 +5,20 @@
   "author": "Happy Prime",
   "license": "GPL-2.0-or-later",
   "devDependencies": {
-    "@wordpress/scripts": "^19.2.2",
-    "@wordpress/stylelint-config": "^19.1.0",
-    "autoprefixer": "^10.4.0",
-    "postcss": "^8.4.5",
+    "@wordpress/scripts": "^22.0.0",
+    "@wordpress/stylelint-config": "^20.0.1",
+    "autoprefixer": "^10.4.2",
+    "postcss": "^8.4.6",
     "postcss-cli": "^9.1.0",
     "postcss-custom-media": "^8.0.0",
```

Comment:
We may not actually need this package - it was probably included courtesy of a lazy copy/paste on my part.